olive/cli/base.py

```diff
@@ -416,7 +416,7 @@ def save_output_model(config: Dict, output_model_dir: Union[str, Path]):
 
     This assumes a single accelerator workflow.
     """
-    run_output_path = Path(config["output_dir"]) / "output_model"
+    run_output_path = Path(config["output_dir"])
```

Comment:
btw without this `output_model` nesting, now the output of the cli would also have the footprint and other files copied over even though they mean nothing to the user of the cli. I think we might need an option to disable saving these files like you mentioned once before.

Reply:
or make it opt-in to save the extra files instead of opt-out. most users only care about the final output.

Reply:
Good call! I don't think users need those files. We should only copy model files here.